dao-utils/src/parts/sequence.rs

```diff
@@ -0,0 +1,27 @@
+use casper_types::U256;
+
+use crate::{instance::Instanced, Variable};
+
+pub struct SequenceGenerator {
+    value: Variable<U256>,
+}
+
+impl SequenceGenerator {
+    pub fn get_current_value(&self) -> U256 {
+        self.value.get()
+    }
+
+    pub fn next_value(&mut self) -> U256 {
+        let next = self.value.get() + U256::one();
+        self.value.set(next);
+        next
+    }
+}
+
+impl Instanced for SequenceGenerator {
```

Comment:
Why this is not derived via macro?

Reply:
Cause the macro uses fully qualified path `casper_dao_utils::instance::Instanced` and if I use it in `casper_dao_utils` the compiler yells at me: `failed to resolve: use of undeclared crate or module `casper_dao_utils`
use of undeclared crate or module `casper_dao_utils``

Reply:
I thought it would work. But that's a good hint on how to organize the code better in Odra.